### PR TITLE
chore: add Next integration example

### PR DESCRIPTION
**Pull request by jonaslagoni**
**Description**
This PR add an example for showing how to integrate Modelina into a Next project.

### CODE REVIEW

File: examples/integrate-with-next/src/pages/index.tsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
+import { getGeneratedCode } from '../modelina';
+import Head from 'next/head';
+import styles from '@/styles/Home.module.css';
+
+function HomePage({ models }: any): any {
```
Comment (kennethaasan) [inline examples/integrate-with-next/src/pages/index.tsx:5]: any reason why this is any? I always question myself whenever I have to use any :D

Comment (jonaslagoni) [reply examples/integrate-with-next/src/pages/index.tsx:5]: Nothing but laziness 😆

Comment (kennethaasan) [reply examples/integrate-with-next/src/pages/index.tsx:5]: I'd like to forbid the use of any in this repo, and only allow exceptions when there's no way around it 😆

Comment (jonaslagoni) [reply examples/integrate-with-next/src/pages/index.tsx:5]: Thats fair, I just thought it didnt matter much for an example 😆 

Let me change it 👍

Comment (kennethaasan) [reply examples/integrate-with-next/src/pages/index.tsx:5]: that's true, but now you have a better example 😄

Comment (jonaslagoni) [reply examples/integrate-with-next/src/pages/index.tsx:5]: That's true 😄

**Review by kennethaasan — APPROVED** [review]

**Comment by jonaslagoni** [comment]
@kennethaasan thanks for the review, however in the CI the example actually fails by some weird dependency error... Something to do with node 16 and below..

**Comment by jonaslagoni** [comment]
We could do it as part of `v2` release I guess? Might as well just upgrade now I guess 🤔

**Comment by kennethaasan** [comment]
> We could do it as part of `v2` release I guess? Might as well just upgrade now I guess 🤔

Sounds good to me!

**Comment by jonaslagoni** [comment]
Continued in https://github.com/asyncapi/modelina/pull/1488

**Pull request closed without merging**